examples/pytorch/language-modeling/README.md

```diff
@@ -14,23 +14,28 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Language modeling 
+# Language modeling training
 
-## language modeling
+The scripts [`run_clm.py`](https://github.com/huggingface/optimum/blob/main/examples/pytorch/language-modeling/run_clm.py) 
+and [`run_mlm.py`](https://github.com/huggingface/optimum/blob/main/examples/pytorch/language-modeling/run_mlm.py)
+allow us to apply different quantization approaches such as dynamic, static and aware-training quantization using 
+different provider such as [Intel Neural Compressor (INC)](https://github.com/intel/neural-compressor) for language 
+modeling tasks.
 
-The script [`run_clm.py`](https://github.com/huggingface/optimum/blob/main/examples/pytorch/language-modeling/run_clm.py), [`run_mlm.py`](https://github.com/huggingface/optimum/blob/main/examples/pytorch/language-modeling/run_mlm.py).
-allows us to apply different quantization approach such as dynamic, post-training and quantization aware-training 
-using different provider such as [Intel Neural Compressor (INC)](https://github.com/intel/neural-compressor) for 
-language modeling tasks.
 
+GPT and GPT-2 are trained or fine-tuned using a causal language modeling (CLM) loss. ALBERT, BERT, DistilBERT and 
+RoBERTa are trained or fine-tuned using a masked language modeling (MLM) loss, more information about the differences 
+between those objectives can be found in our [model summary](https://huggingface.co/transformers/model_summary.html).
 
-The following example applies dynamic quantization on a distilroberta fine-tuned on the wikitext datasets, using the
-[`inc`](https://github.com/intel/neural-compressor) provider: 
 
+### GPT-2/GPT and causal language modeling
+
+The following example fine-tunes GPT-2 on WikiText-2 while applying [`inc`](https://github.com/intel/neural-compressor) 
+quantization aware training. We're using the raw WikiText-2 (no tokens were replaced before the tokenization). The loss here is that of causal language modeling (CLM). 
 
 ```bash
-python examples/pytorch/language-modeling/run_clm.py \
-    --model_name_or_path bert-base-cased \
+python run_clm.py \
+    --model_name_or_path gpt2 \
```

Comment:
For GPT2 model, the customized Conv1d ops are not in the PyTorch quantization map, so only one layer of linear will be quantized in PyTorch FBGEMM. So the benefits of quantization are relatively small for this model. I think it is not a good example here, do you think so?

Reply:
I see your point, as CLM is unidirectional in nature, what do you think about replacing it by a GPT-Neo model (such as EleutherAI/gpt-neo-125M for example) ?

Reply:
Yes, I tested EleutherAI/gpt-neo-125M, and it is ok for me. I modified README and quantization.yml, please review the below commit:
https://github.com/huggingface/optimum/pull/30/commits/128d4573223ae11280262c21b71b7dbfccecdc9a